reader-osm/pom.xml

```diff
@@ -45,17 +45,44 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.12</version>
+            <groupId>org.junit.platform</groupId>
```

Comment:
Maybe we could put these dependencies into the parent pom?

Reply:
Ok